subsys/net/l2/canbus/Kconfig

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config NET_L2_CANBUS
+	bool "Enable CANBUS L2 layer [EXPERIMENTAL]"
+	depends on CAN_NET
+	select NET_6LO
+	help
+	  Add a CANBUS L2 layer driver. This is the layer for IPv6 over CAN
+	  (6loCAN). It uses IPHC to compress the IP header and ISO-TP for
+	  flow control and reassembling.
+
+if NET_L2_CANBUS
+
+config NET_L2_CANBUS_USE_FIXED_ADDR
+	bool "Use fixed L2 address"
+	help
+	  Use a fixed L2 address for 6LoCAN instead of a random chosen one
+
+config NET_L2_CANBUS_FIXED_ADDR
+	hex "L2 address"
+	depends on NET_L2_CANBUS_USE_FIXED_ADDR
+	range 0x00FF 0x3DEF
+
+config NET_L2_CANBUS_DAD_RETRIES
+	int "Number of DAD retries"
+	default 5
+	help
+	  Number of retries for Duplicate Address Detection.
+	  Greater than one only makes sense for random link layer addresses.
+
+config NET_L2_CANBUS_STMIN
+	int "STmin"
```

Comment:
I'd suggest to definitely use more detailed description, and perhaps use more detailed option name.